superbench/analyzer/data_diagnosis.py

```diff
@@ -31,10 +31,15 @@ def _get_metrics_by_benchmarks(self, metrics_list):
         """
         benchmarks_metrics = {}
         for metric in metrics_list:
-            benchmark = metric.split('/')[0]
-            if benchmark not in benchmarks_metrics:
-                benchmarks_metrics[benchmark] = set()
-            benchmarks_metrics[benchmark].add(metric)
+            if '/' not in metric:
+                if 'monitor' not in benchmarks_metrics:
```

Comment:
why not use 'monitor/' prefix for monitor metrics?

Reply:
I agree. I think we should fix in the monitor.

Reply:
Solved.